change/@azure-msal-angular-db641b59-aad7-49f5-8b4f-06625de69bab.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Add support for requested claims in silent token acquisition #4296",
+  "packageName": "@azure/msal-angular",
+  "email": "hectormg@hey.com",
```

Comment:
Double check your git config on a couple of these

Reply:
Thanks, changed the incorrect e-mails.